### ISSUE TITLE
Export the same value under multiple names?

### ISSUE THREAD

**Issue by mbostock** [rollup/rollup#18]
I should be able to do this, right?

```js
import {bisectRight, bisectLeft} from "./src/bisect";

export {
  bisectLeft,
  bisectRight,
  bisectRight as bisect
}
```

**Comment by Rich-Harris** [comment]
I haven't tried it but yes, if you can't do that then I think it's a bug.

**Comment by Rich-Harris** [comment]
Yep, definite bug. Not immediately sure what the most elegant solution looks like, so for now I've just added a failing test - #19

**Comment by Rich-Harris** [comment]
This is fixed in 0.14.1. (Actually I think it was probably fixed earlier, but 0.14.1 extends the fix to related cases.)
